pgx/Cargo.toml

```diff
@@ -33,26 +33,33 @@ no-default-features = true
 rustc-args = ["--cfg", "docsrs"]
 
 [dependencies]
-bitflags = "1.3.2"
-bitvec = "1.0" # processing array nullbitmaps
-cstr_core = "0.2.5"
 enum-primitive-derive = "0.2.2"
 num-traits = "0.2.15"
-seahash = "4.1.0"
-libc = "0.2.126"
+
```

Comment:
Add a note on line 36 to make sure that developers give a reason for the addition.

Reply:
I think we should add a note to CONTRIBUTING.md instead since there's multiple Cargo.tomls in the repo and this PR only sorts/annotates one. I think the "peer pressure" of seeing the small notes will assist in enforcing this norm in-location.